### PR TITLE
feat: Implement WQuickSecurityContextManagerV1

### PR DESCRIPTION
**Pull request by lychee-yycy**

### CODE REVIEW

File: src/server/qtquick/private/wquicksecuritycontextmanagerv1_p.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#include <wglobal.h>
+#include <wquickwaylandserver.h>
+#include <qwglobal.h>
+
+#include <QObject>
+#include <QQmlEngine>
+
+WAYLIB_SERVER_BEGIN_NAMESPACE
+
+    class WQuickSecurityContextManagerV1Private;
```
Comment (zorowk) [inline src/server/qtquick/private/wquicksecuritycontextmanagerv1_p.h:15]: indent is not aligned

Comment (lychee-yycy) [reply src/server/qtquick/private/wquicksecuritycontextmanagerv1_p.h]: done

File: src/server/qtquick/private/wquickfractionalscalemanagerv1.cpp [hunk 2]
```diff
@@ -30,7 +30,7 @@ WQuickFractionalScaleManagerV1::WQuickFractionalScaleManagerV1(QObject *parent)
     , WObject(*new WQuickFractionalScaleManagerV1Private(this), nullptr)
 {
     if (FRACTIONAL_SCALE_MANAGER) {
-        qFatal("There are multiple instances of WQuickForeignToplevelManagerV1");
+        qFatal("There are multiple instances of WQuickFractionalScaleManagerV1");
```
Comment (zccrs) [inline src/server/qtquick/private/wquickfractionalscalemanagerv1.cpp:33]: Fix in a new commit.

**Review by zccrs — CHANGES_REQUESTED** [review]

**Comment by lychee-yycy** [comment]
qwlroots refactored, abandoned

**Pull request closed without merging**